syntax = "proto3";

package sentient;

option java_package = "sentient";
option java_outer_classname = "CSVProtos";

message ParseCsvRequest {
  string csvPath   = 1;
  int32 outputPort = 2;
}

message ParseCsvResponse {
  int32 outputPort = 1;
}

message Row {
  repeated string field = 1;
}
